frontend/web/components/base/forms/Checkbox.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+import Icon from 'components/Icon'
+
+interface CheckboxProps {
+  label: string
+  onChange: (value: boolean) => void
+  checked: boolean
+}
+
+const Checkbox: React.FC<CheckboxProps> = ({ checked, label, onChange }) => {
+  const handleChange = () => {
+    onChange(!checked)
+  }
+
+  return (
+    <>
+      <input type='checkbox' />
+      <label
+        onClick={handleChange}
+        className='mb-0'
+        htmlFor='mailinglist'
```

Comment:
I think I understand most of this file, but I don't understand why the htmlFor would point to mailinglist.

Reply:
After reading the rest of the code I understand that its being used on the home page in an email list. It would be better if something as reusable as a checkbox was less hardcoded to its use elsewhere.